packages/forge/blocks/nocodb/actions/listTableRecords.ts

```diff
@@ -0,0 +1,105 @@
+import { createAction, option } from '@typebot.io/forge'
+import { auth } from '../auth'
+import { HTTPError, got } from 'got'
+import { ListTableRecordsResponse } from '../types'
+import { isDefined } from '@typebot.io/lib'
+
+export const listTableRecords = createAction({
+  auth,
+  name: 'List Table Records',
+  options: option.object({
+    tableId: option.string.layout({
+      label: 'Table ID',
+      isRequired: true,
+      helperText: 'Identifier of the table to retrieve records from.',
```

Comment:
I'm not a fan of having a helper text for each input.

Here is the protocol I follow:
- use 'moreInfoTooltip' to provide more info if the label is ambiguous
- use 'helperText' to provide information on how to retrieve/look for that field